bylaws/5-decision-making.md

```diff
@@ -37,34 +33,35 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
 
 ### Sense Vote
 
+Members express their their level of comfort or agreement with proposals using *reactions*.
+
+Sense votes typically use the following reactions, though alternatives may be suggested for specific proposals:
```

Comment:
Sense votes?

Reply:
That is terminology from Martha's rules. It is not (strictly speaking) a formal vote. But if there is consensus, you can move forward.